smarkets/streaming_api/framing.py

```diff
@@ -17,43 +17,43 @@ def frame_encode(payload):
     return header + payload + padding
 
```

Comment:
@AdamStelmaszczyk Since we are returning `bytearray`, wouldn't it make more sense to avoid the uses of `+` and instead do something like:
```python
byte_count = len(payload)
header = uleb128_encode(byte_count)
padding = b'\x00' * max(0, 4 - len(header) - byte_count)
frame = bytearray()
frame += header
frame += payload
frame += padding
return frame
```